dev/com.ibm.ws.security.common/src/com/ibm/ws/security/common/web/JavaScriptUtils.java

```diff
@@ -229,6 +229,9 @@ public Map<String, String> getWebAppSecurityConfigCookieProperties(WebAppSecurit
                     cookieProperties.put("secure", null);
                 }
             }
+            if (webAppSecurityConfig.getPartitionedCookie()) {
+              cookieProperties.put("Partitioned", null);
+            }
```

Comment:
Since the `Partitioned` property should only be set when `SameSite=None`, shouldn't this be moved two lines up? e.g.:
```Java
if ("None".equalsIgnoreCase(sameSite)) {
    cookieProperties.put("secure", null);
    if (webAppSecurityConfig.getPartitionedCookie()) {
        cookieProperties.put("Partitioned", null);
    }
}
```

Reply:
well...that is from external.  WC wants us to pass out the partitioned value regardless of the SS value in case SS was set at their level.  We only do the absolute checking in the JS code.  I did it the way you said the first time and WC asked me to move it.